### PR TITLE
update Mac FAQs

### PR DESCRIPTION
**Pull request by usha-mandya**
Fixes https://docker.atlassian.net/browse/ENGDOCS-194

This PR contains the following changes:

- Updated Docker for Mac references to Docker Desktop
- Added Stable and Edge download info
- Added a new question about nested virtualization
- Moved info about switching between Edge and Stable and instructions on how to save and restore data to the **Install** topic
- Removed information about Disk utilization and instructions on pruning as this info is already present in https://docs.docker.com/docker-for-mac/space/
- Removed specific instructions on adding client certificates and added redirection to the detailed info in the Getting started topic

### CODE REVIEW

**Comment by usha-mandya** [comment]
@nebuk89 @djs55 could you PTAL - https://deploy-preview-9596--docsdocker.netlify.com/docker-for-mac/faqs/

File: docker-for-mac/faqs.md [hunk 1]
```diff
@@ -6,112 +6,42 @@ redirect_from:
...
-    [load](/engine/reference/commandline/load) in the Docker Engine
-
-6.  Use `docker import -i myContainer1.tar` to create a filesystem image
-    corresponding to previously exported containers. (See
-    [import](/engine/reference/commandline/import) in the Docker Engine
-
-[This
-procedure](https://docs.docker.com/storage/volumes/#backup-restore-or-migrate-data-volumes)
-explains how to backup and restore data volumes.
+A: Yes, you can switch between Stable and Edge versions. You can try out the Edge releases to see what's new, then go back to Stable for other work. However, **you can only have one version of Docker Desktop installed at a time**. For more information, see [Switch between Stable and Edge versions](/docker-for-mac/install/#switch-between-stable-and-edge-versions).
```
Comment (traci-morrison) [inline docker-for-mac/faqs.md]: s/Switch/switch

Comment (traci-morrison) [inline docker-for-mac/faqs.md]: s/Switch/switch

File: docker-for-mac/faqs.md [hunk 2]
```diff
@@ -123,18 +53,15 @@ To make the client talk to the Docker for Mac Engine, run the command `unset
...
-to talk to Docker for Mac. The same applies to `docker-compose`.
+to talk to Docker Desktop. The same applies to `docker-compose`.
 
 ### How do I uninstall Docker Toolbox?
 
-You might decide that you do not need Toolbox now that you have Docker for Mac,
-and want to uninstall it. For details on how to perform a clean uninstall of
-Toolbox on the Mac, see [How to uninstall Toolbox](/toolbox/toolbox_install_mac/#how-to-uninstall-toolbox)
-in the Toolbox Mac topics.
+Now that you have installed Docker Desktop, you might decide that you do not need the legacy Docker Toolbox anymore. For details on how to perform a clean uninstall of Toolbox on Mac, see [How to uninstall Toolbox](/toolbox/toolbox_install_mac/#how-to-uninstall-toolbox).
```
Comment (traci-morrison) [inline docker-for-mac/faqs.md]: s/How/how

File: docker-for-mac/faqs.md [hunk 3]
```diff
@@ -220,130 +131,40 @@ export DOCKER_HOST=unix:///var/run/docker.sock
 
 ### How do I connect from a container to a service on the host?
 
-The Mac has a changing IP address (or none if you have no network access). Our
-current recommendation is to attach an unused IP to the `lo0` interface on the
+Mac has a changing IP address (or none if you have no network access). We recommend that you attach an unused IP to the `lo0` interface on the
```
Comment (traci-morrison) [inline docker-for-mac/faqs.md]: Is part of this sentence missing?

File: docker-for-mac/faqs.md [hunk 4]
```diff
@@ -220,130 +131,40 @@ export DOCKER_HOST=unix:///var/run/docker.sock
...
-
-Note the `Docker.qcow2` does not shrink in size immediately.
-In 1.13 a background `cron` job runs `fstrim` every 15 minutes.
-If the space needs to be reclaimed sooner, run this command:
-
-```bash
-$ docker run --rm -it --privileged --pid=host walkerlee/nsenter -t 1 -m -u -i -n fstrim /var
-```
+Unfortunately, it is not possible to pass through a USB device (or a
+serial port) to a container.
```
Comment (traci-morrison) [inline docker-for-mac/faqs.md]: Should we state why this is not possible or what the alternate/preferred way is?

File: docker-for-mac/install.md [hunk 5]
```diff
@@ -76,6 +76,41 @@ The Docker Desktop installation includes
 
 Congratulations! You are now successfully running Docker Desktop.
 
+## Switch between Stable and Edge versions
+
+Docker Desktop allows you to switch between Stable and Edge releases. However, **you can only have one version of Docker Desktop installed at a time**. Switching between Stable and Edge versions can destabilize your development environment, particularly in cases where you switch from a newer (Edge) channel to older (Stable).
+
```
Comment (traci-morrison) [inline docker-for-mac/install.md]: s/channel to older/channel to an older channel

File: docker-for-mac/faqs.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/faqs.md]: The actual section heading is Switch between Stable and Edge versions - https://deploy-preview-9596--docsdocker.netlify.com/docker-for-mac/install/#switch-between-stable-and-edge-versions

File: docker-for-mac/install.md [hunk 6]
```diff
@@ -76,6 +76,41 @@ The Docker Desktop installation includes
...
+For example, containers created with a newer Edge version of Docker Desktop may
+not work after you switch back to Stable because they may have been created
+using Edge features that aren't in Stable yet. Just keep this in mind as
```
Comment (traci-morrison) [inline docker-for-mac/install.md]: s/Just keep/Please keep

File: docker-for-mac/install.md [hunk 7]
```diff
@@ -76,6 +76,41 @@ The Docker Desktop installation includes
...
+
+To safely switch between Edge and Stable versions, ensure you save images and export the containers you need, then uninstall the current version before installing another. For more information, see the section Save and Restore data below.
+
+### Save and restore data
+
+You can use the following procedure to save and restore images and container data. For example, if you want to switch between Edge and Stable, or to reset your VM disk:
+
+1. Use `docker save -o images.tar image1 [image2 ...]` to save any images you
+    want to keep. (See [save](/engine/reference/commandline/save) in the Docker
+    Engine command line reference.)
```
Comment (traci-morrison) [inline docker-for-mac/install.md]: I would remove the () around the second sentence. This occurs in a few places.

File: docker-for-mac/install.md [hunk 8]
```diff
@@ -76,6 +76,41 @@ The Docker Desktop installation includes
...
+3. Uninstall the current version of Docker Desktop and install a different version (Stable or Edge), or reset your VM disk.
+
+4. Use `docker load -i images.tar` to reload previously saved images. (See
+    [load](/engine/reference/commandline/load) in the Docker Engine.
+
+5. Use `docker import -i myContainer1.tar` to create a filesystem image
+    corresponding to the previously exported containers. (See
+    [import](/engine/reference/commandline/import) in the Docker Engine.
+
+For information on how to back up and restore data volumes, see [Backup, restore, or migrate data volumes](/storage/volumes/#backup-restore-or-migrate-data-volumes).
```
Comment (traci-morrison) [inline docker-for-mac/install.md]: s/Backup/backup

File: docker-for-mac/faqs.md (hunk 3 above)
Comment (usha-mandya) [reply docker-for-mac/faqs.md]: No, it is not. Git only displays the delta. Here's the full info - https://deploy-preview-9596--docsdocker.netlify.com/docker-for-mac/faqs/#how-do-i-connect-from-a-container-to-a-service-on-the-host

File: docker-for-mac/install.md (hunk 5 above)
Comment (usha-mandya) [reply docker-for-mac/install.md]: Agree. Will update accordingly.

File: docker-for-mac/install.md (hunk 7 above)
Comment (usha-mandya) [reply docker-for-mac/install.md]: Agree. Will update this section.

File: docker-for-mac/faqs.md (hunk 4 above)
Comment (usha-mandya) [reply docker-for-mac/faqs.md]: Good point. Check with Mathieu on this. I'll update when we have any update.

Comment (usha-mandya) [reply docker-for-mac/faqs.md]: Updated to `Unfortunately, it is not possible to pass through a USB device (or a serial port) to a container as it requires support at the hypervisor level.`

File: docker-for-mac/install.md (hunk 5 above)
Comment (usha-mandya) [reply docker-for-mac/install.md]: Fixed

File: docker-for-mac/install.md (hunk 7 above)
Comment (usha-mandya) [reply docker-for-mac/install.md]: Fixed

File: docker-for-mac/install.md (hunk 6 above)
Comment (usha-mandya) [reply docker-for-mac/install.md]: Fixed

**Pull request merged**